src/vs/base/parts/tree/browser/treeModel.ts

```diff
@@ -980,19 +980,28 @@ export class TreeModel {
 	}
 
 	public refresh(element: any = null, recursive: boolean = true): WinJS.Promise {
-		var item = this.getItem(element);
+		var item: Item = this.getItem(element);
```

Comment:
is this really needed?

Reply:
It is helpful for other dev to recognize the object is not just `any` type. Currently a lot of objects are used as `any` type blindly, and it is very hard for other dev to know what member field and methods are belongs to the object explicitly.